src/README.md

```diff
@@ -0,0 +1 @@
+# MDM API
```

Comment:
We already have a root level README file. I'm not sure we need this here?

Reply:
Good shout, will remove.